facefusion/processors/modules/background_remover.py

```diff
@@ -0,0 +1,167 @@
+from argparse import ArgumentParser
+from functools import lru_cache
+from typing import List
+
+import cv2
+import numpy
+
+import facefusion.jobs.job_manager
+import facefusion.jobs.job_store
+from facefusion import config, content_analyser, inference_manager, logger, state_manager, video_manager, wording
+from facefusion.common_helper import is_macos
+from facefusion.download import conditional_download_hashes, conditional_download_sources
+from facefusion.execution import has_execution_provider
+from facefusion.filesystem import in_directory, is_image, is_video, resolve_relative_path, same_file_extension
+from facefusion.processors import choices as processors_choices
+from facefusion.processors.types import FrameColorizerInputs
+from facefusion.program_helper import find_argument_group
+from facefusion.thread_helper import thread_semaphore
+from facefusion.types import ApplyStateItem, Args, DownloadScope, ExecutionProvider, InferencePool, Mask, ModelOptions, ModelSet, ProcessMode, VisionFrame
+from facefusion.vision import read_static_image, read_static_video_frame
+
+
+@lru_cache()
+def create_static_model_set(download_scope : DownloadScope) -> ModelSet:
+	return\
+	{
+		'birefnet-general-244':
+		{
+			'hashes':
+			{
+				'background_remover':
+				{
+					'url': 'https://huggingface.co/bluefoxcreation/background-removers/resolve/main/BiRefNet-general-epoch_244.hash',
+					'path': resolve_relative_path('../.assets/models/BiRefNet-general-epoch_244.hash')
+				}
+			},
+			'sources':
+			{
+				'background_remover':
+				{
+					'url': 'https://huggingface.co/bluefoxcreation/background-removers/resolve/main/BiRefNet-general-epoch_244.onnx',
+					'path': resolve_relative_path('../.assets/models/BiRefNet-general-epoch_244.onnx')
+				}
+			},
+			'size': (1024, 1024),
+			'mean': [ 0.0, 0.0, 0.0 ],
```

Comment:
how sure are you that we need mean and standard_deviation in this processor?

Reply:
rembg need different mean and std

Reply:
ok